src/model.rs

```diff
@@ -31,20 +31,49 @@ pub struct Item {
     pub thumbnail_url: Url,
     pub apply_matte: bool,
     pub activate_by: String,
+    pub predominant_color: Option<String>,
 }
 
 #[derive(Debug, Clone, Serialize, Deserialize)]
-pub struct Played {
+#[serde(untagged)]
+#[serde(rename_all = "camelCase")]
+pub enum ChannelContent {
+    ChannelContentV1 {
+        items: Vec<Item>,
+        played: Played,
+    },
+    ChannelContentV2 {
+        items: Vec<Item>,
+        #[serde(default = "default_item_duration")]
+        item_duration: u32,
+        status: Status,
+    },
+}
+
+// ChannelContentV2
+
+// Function to provide the default value
+fn default_item_duration() -> u32 {
+    60
+}
+
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub struct Status {
     pub item: u32,
     pub at: DateTime<Utc>,
+    pub action: String,
```

Comment:
I think this should also be an enum, just to keep in line with Rust's strict typing. We can always easily extend it still, without a new version anyway.